src/contracts/middleware/BLSSignatureChecker.sol

```diff
@@ -154,6 +185,16 @@ abstract contract BLSSignatureChecker {
         return (quorumStakeTotals, signatoryRecordHash);
     }
 
+    /// @return count number of ones in binary representation of `n`
+    function countNumOnes(uint256 n) public pure returns (uint16) {
+        uint16 count = 0;
+        while (n > 0) {
+            n &= (n - 1);
+            count++;
+        }
+        return count;
+    }
+
```

Comment:
isn't this in the BitmapUtils library now? seems like code duplication.

Reply:
yes!, thought we were going to merge these at different times. https://github.com/Layr-Labs/eigenlayer-contracts/pull/81/commits/4b9988c1fa57592bda1cab50540f32831fc09310